### ISSUE TITLE
Stat on empty dir yields unwrap() error

### ISSUE THREAD

**Issue by robklg** [bolcom/libunftp#418]
Got this error with Cyberduck after creating an empty dir and trying to open it.

```
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /Users/rkleingunnewiek/unFTP/libunftp/src/server/controlchan/codecs.rs:61:45
```

Cyberduck uses `STAT` to list directory contents. This makes sense cause it avoids the need for LIST (and a separate data channel). This needs to return a multi line response. But the encoder  doesn't expect an empty directory. It does an unwrap() on an Option. https://github.com/bolcom/libunftp/blob/master/src/server/controlchan/codecs.rs#L61

**Comment by robklg** [comment]
To reproduce:

```
2022-06-20 22:32:28 ---> stat /emptydir
2022-06-20 22:32:28 **** Peer closed connection 
2022-06-20 22:32:28 ---- Closing control socket
`stat /emptydir' at 0 [Delaying before reconnect: 66]   
```

**Comment by hannesdejager** [comment]
Fixed in v0.18.5
